docs/team/README.md

```diff
@@ -27,233 +27,236 @@ Then, click on the ![Add a new user](../img/getting-started/add_employee.png) bu
 
-The first page of an Artist in Kitsu is their todo page.
+::: warning Definition
+A permission role defines a set of access rights and privileges granted to a user within a system or application, dictating what actions they can perform and what resources they can access.
+:::
 
-![my task](../img/getting-started/my_task_page.png)
+Roles are very important, so it's useful to understand what each of them does and which ones might be relevant to specific team members. Click into each of the sections below to learn more about each permission role.
 
+- **Artist**
+::: details Artist Permissions
+Artists can only access the productions they are part of. They can comment on tasks, upload media, and change statuses only on tasks that have been assigned to them. Their access is limited to a predefined set of statuses as determined by the Studio Manager.
 
-::: tip
-The Artist has access to all the features of the production, but only for display.
+**They can:**
+* Create personal filters on the global page and Task Type page.
+* Edit their own comments.
+* Check the checklist on their assigned tasks.
+* Create playlists-on-the-fly for shots or assets, but won't be able to save these playlists.
+
+**They cannot:**
+* See client comments.
+* Access anything inside of projects that they haven't been assigned to.
 
+When an artist logs in to Kitsu, the first page they will see is their **My Tasks** page.
 
+![my task](../img/getting-started/my_task_page.png)
 :::
 
 - **Supervisor**
-::: details Department Supervisors Permission
+::: details Supervisor Permissions
 Department supervisors inherit Artist permissions.
 
 Department supervisors have read and write access to their department(s) they work on:
-assets, shots, tasks, assignations, statistics, breakdown, and playlists.
-
-They can:
-
-* Assign task to their team artist (same department)
-* Post comment on all tasks or their department(s)
-* Check a checklist on their own department
-* Pin a comment
-* Edit its own comment
+assets, shots, tasks, assignments, statistics, breakdown, and playlists.
+
+**They can:**
+* Assign tasks to their team artists (same department).
+* Post comments on all tasks or their department(s).
+* Check a checklist in their own department.
+* Pin a comment.
+* Edit their own comments.
 * Add/edit a playlist for the studio or the client.
-* See the client comments and validations.
-* See comment on other departments.
-* See the timesheets of his/her team departement(s)
-
-
-They are not allowed to access the studio team, the main timesheets, and the
-production list. They can't define task types, task statuses, and asset types
-neither.
-they can't comment on other departments than theirs, they can't assign artist from other department.
+* See client comments and validations.
+* See comments from other departments.
+* View the timesheets of their team department(s).
+
+**They cannot:**
+* Access the studio team, the main timesheets, and the
```

Comment:
It seems that the end of the phrase is missing.

Reply:
Thanks, this has been addressed